sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/HISTORY.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 2019-12-04 1.0.0b6
+
+**Breaking changes**
+
+- Renamed `BlobPartitionManager` to `BlobCheckpointStore`.
+- Constructor of `BlobCheckpointStore` has been updated to take the storage container details directly rather than an instance of `ContainerClient`.
+- A `from_connection_string` constructor has been added for Blob Storage connection strings.
+- Module `blobstoragepmaio` is now internal, import directly from `azure.eventhub.extensions.checkpointstoreblobaio`.
```

Comment:
Should users be importing it, if it is internal?

Reply:
That's poor wording on my part - I meant to state that if users were importing anything from that module, they should now import directly from checkpointstoreblob instead.....
I will revise.

Reply:
Updated